crates/viewer/re_viewer_context/src/store_hub.rs

```diff
@@ -391,6 +396,7 @@ impl StoreHub {
     }
 
     /// Remove all open recordings and applications, and go to the welcome page.
+    // TODO(grtlr): Make sure we also clear all tables
```

Comment:
Maybe this should link to the issue
* https://github.com/rerun-io/rerun/issues/9465